eng/common/build.ps1

```diff
@@ -86,6 +86,10 @@ function InitializeDotNetCli {
     $env:DOTNET_INSTALL_DIR = $dotnetRoot
     
     if ($restore) {
+      # We need .NET Core 1.1.2 for executing unit tests
+      InstallDotNetRuntime $dotnetRoot 1.1.2
```

Comment:
@tmat just letting you know we are doing this

Reply:
Move this to `RestoreToolset.ps1`. See https://github.com/dotnet/cli-migrate/blob/master/build/RestoreToolset.ps1